clients/python/migrate.py

```diff
@@ -33,22 +33,25 @@ def main():
     client = SysAdminClient('localhost', args.port)
     migrator = SysAdminMigrator(client)
     migration_history = []
-    migrations = load_migrations(args.migration_file)
-    for i, m in enumerate(migrations):
-        if migration_log.check_history(m[1]):
-            print("Migration %s has already run, skipping..." % m[0])
-            continue
-        print("Running migration %s" % (m[0]))
-        resp = migrator.migrate(dict(m[1]))
-        migration_history.append((m[1], resp.commit.commit_id))
-        if resp.status != sysadminctl_pb2.SUCCESS and i == len(m) - 1:
-            print("Migration failed, quitting and rolling back")
-            migration_history.reverse()
-            for migration, cid in migration_history:
-                client.rollback(cid)
-                migration_log.redo_migration(migration)
-            sys.exit(1)
-        migration_log.log_migration(m[1])
+    try:
```

Comment:
It's not that this is wrong, per se, but it feels a bit heavy handed. What are you trying to catch here?

Reply:
We discussed this offline. This is being removed as _I think_ there is a bug if one of the migrations fails, the ones that didn't run will get logged anyway.